cmd/aws-vpc-cni/main_test.go

```diff
@@ -48,6 +49,33 @@ func TestGenerateJSONPlusBandwidthAndTuning(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+// Validate setting environment AWS_VPC_ENI_MTU, takes effect for egress-cni plugin
+func TestEgressCNIPlugin(t *testing.T) {
```

Comment:
Test function appears to be validating the MTU value passed to the egress CNI plugin and not the plugin functionality itself, so I would suggest renaming the function to reflect that

Reply:
Yeah, it was covering this change only 
https://github.com/aws/amazon-vpc-cni-k8s/commit/fe4c7a1d8b8dd6c73b84338825a762c739df472e